string-manipulation-3/src/is-anagram.js

```diff
@@ -0,0 +1,16 @@
+/* exported isAnagram */
+function isAnagram(firstString, secondString) {
+  var firstStringArray = firstString.split('');
+  var secondStringArray = secondString.split('');
+  firstStringArray.sort();
+  secondStringArray.sort();
+  firstString = firstStringArray.join('');
+  secondString = secondStringArray.join('');
+  firstString = firstString.trim();
+  secondString = secondString.trim();
+  if (secondString === firstString) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
If you are just returning  true or false from a comparison you can just return the comparison, the comparison will be evaluated to true or false then that value will be returned.

```JAVASCRIPT
return secondString === firstString;
```